h2o-bindings/bin/gen_java.py

```diff
@@ -637,12 +669,16 @@ def generate_main_class(endpoints):
             try {
               int i = 0;
               for (String id: ids) {
-                keys[i] = clz.newInstance();
+                keys[i] = clz.getConstructor().newInstance();
```

Comment:
One more curious question: who is loading clz? it is always good to know the classloader, especially in this case when we are expecting people to embed the generated code into their applications.